tests/_support/Page/Widget/Grid.php

```diff
@@ -120,6 +121,35 @@ public function getRowDataKeyByNumber(int $rowNumber): string
         return $this->tester->grabAttributeFrom($selector, 'data-key');
     }
 
+    public function ensureSeeValueInColumn($columnNumber, $tableValue): void
+    {
+        $this->tester->see($tableValue, "//table//tbody//td[$columnNumber]//a[contains(text(), '$tableValue')]");
+    }
+
+    public function getColumnNumber(string $columnName): int
+    {
+        $columnNumber = 2;
+        $I = $this->tester;
+        $headElements = $I->grabMultiple('//th[not(./input)]');
+        foreach ($headElements as $currentColummName) {
+            if($columnName == $currentColummName) {
```

Comment:
```suggestion
            if($columnName === $currentColummName) {
```
почитай про різницю в пхп `==` і `===`